src/modules/Dynamic/Module.js

```diff
@@ -25,48 +25,67 @@ module.exports = function Dynamic(options,UI) {
       eval(func);
       return f;
     }
-
+    
     var channels = ['red', 'green', 'blue', 'alpha'];
-
+    
     channels.forEach(function(channel) {
       if (options.hasOwnProperty(channel)) options[channel + '_function'] = generator(options[channel]);
       else if (channel === 'alpha')        options['alpha_function'] = function() { return 255; }
       else                                 options[channel + '_function'] = generator(options.monochrome);
     });
-
+    
     function changePixel(r, g, b, a) {
+
+      /* neighbourpixels can be calculated by
+      
+       this.getNeighbourPixel.fun(x,y)  or this.getNeighborPixel.fun(x,y)
+       
+       */
       var combined = (r + g + b) / 3.000;
       return [
-        options.red_function(  r, g, b, a),
+        options.red_function(r, g, b, a),
         options.green_function(r, g, b, a),
-        options.blue_function( r, g, b, a),
+        options.blue_function(r, g, b, a),
         options.alpha_function(r, g, b, a),
       ];
     }
-
+    
+    /* Functions to get the neighbouring pixel by position (x,y) */
+    var getNeighbourPixel = function(pixels,curX,curY,distX,distY){
```

Comment:
Hmm, yes, this does seem a bit more natural! You can also define this as `function getNeighborPixel(...)` which helps when debugging (the function name shows up rather than a singleton function assigned to a variable). If this looks good, can we tidy up lines 38-43 and we'll be ready to merge it in? Thanks!!!